### PR TITLE
[REF] 8.0 mgmtsystem nonconformity claim

### PR DESCRIPTION
**Pull request by llacroix**
Marked as installable and no other change were required for the port. Doesn't yet use API version 8
- [x] Depends on #25

### CODE REVIEW

**Comment by bwrsandman** [comment]
Please:
- Split `mgmtsystem_nonconformity.py` to `mgmtsystem_nonconformity.py` and `mgmtsystem_nonconformity_type.py`
- Split `mgmtsystem_nonconformity.xml` to `mgmtsystem_nonconformity_view.xml` and `mgmtsystem_nonconformity_type_view.xml`
- Rename `mgmtsystem_nonconformity_data.xml` to `mgmtsystem_nonconformity_type_data.xml`

File: mgmtsystem_nonconformity_claim/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline mgmtsystem_nonconformity_claim/__init__.py]: coding

File: mgmtsystem_nonconformity_claim/__openerp__.py [hunk 2]
```diff
@@ -36,12 +36,14 @@
   * a common numbering sequence for complaints and nonconformities;
   * a single point-of-entry for all management system related occurrences.
 """,
```
Comment (dreispt) [inline mgmtsystem_nonconformity_claim/__openerp__.py]: Move description to a README

File: mgmtsystem_nonconformity_claim/__init__.py (hunk 1 above)
Comment (bwrsandman) [inline mgmtsystem_nonconformity_claim/__init__.py]: encoding works too

**Comment by bwrsandman** [comment]
openerp.addons.mgmtsystem_nonconformity.tests.test_cause: test_create_cause is failing due to a not null constraint, this could be fixed by adding a `name` or it could be the fact that another module added a SQL level constraint, which is tougher to fix

File: mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py [hunk 3]
```diff
@@ -35,12 +35,3 @@ class mgmtsystem_nonconformity_type(orm.Model):
...
     }
-
-
-class mgmtsystem_nonconformity(orm.Model):
-    _name = "mgmtsystem.nonconformity"
-    _inherit = "mgmtsystem.nonconformity"
-    _description = "Feedback and Nonconformities"
-    _columns = {
-        'type_id': fields.many2one('mgmtsystem.nonconformity.type', 'Type'),
-    }
```
Comment (dreispt) [inline mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py:46]: Is there a reason to remove this feature?

Comment (bwrsandman) [inline mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py:46]: Not removed, just split.

See https://github.com/OCA/management-system/pull/33/files#diff-13471760dcc9d006b4e0e2f47672bfc4R24

Comment (dreispt) [inline mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py:46]: Wrong link?

Comment (max3903) [inline mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py:46]: Link is good. It is pointing to another file with this code.

Comment (dreispt) [inline mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py:46]: Of course it is. I was twice blind.

**Comment by dreispt** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**